### DISCUSSION TITLE
spike prime unexpected crash

### DISCUSSION THREAD

**Discussion by ikengo** [pybricks/support#1597] [Q&A]
I plan to use Pybricks to compete in FLL.

I select multiple programs in one program. However, Spike crashes when changing attachments. I think vibration is the cause. Please let me know if there is a workaround. please.

**Comment by BertLindeman** [comment]
Might your problem be related to #1388 

Vibration does let me think it might be a loose battery.

If not, it might help if you explain a bit more about the situation.

Bert

**Comment by laurensvalk** [comment]
What do you mean by crash? Do you see errors? Does the brick turn off? 

Some Spike Prime hubs are known to have a very loose battery (unrelated to software), which tends to move around and lose contact so it turns off. Some teams have fixed it by adding a piece of cardboard on the other side to make sure the battery stays pushed against the contacts.
